artifacts/deploy/kube-controller-manager.yaml

```diff
@@ -24,63 +24,62 @@ spec:
       affinity:
         podAntiAffinity:
           requiredDuringSchedulingIgnoredDuringExecution:
-            - labelSelector:
-                matchExpressions:
-                  - key: app
-                    operator: In
-                    values:
-                      - kube-controller-manager
-              topologyKey: kubernetes.io/hostname
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
```

Comment:
Nice finding. 
But I suggest don't fix the format issue in this PR since it will be picked to release branches, otherwise, we need to solve the conflicts. 
No big deal though.

Reply:
Sorry, Caused by the format plugins. I will modify it back

Reply:
done